src/pybind/mgr/ansible/run-tox.sh

```diff
@@ -20,10 +27,15 @@ export CEPH_BUILD_DIR=$CEPH_BUILD_DIR
 source ${MGR_ANSIBLE_VIRTUALENV}/bin/activate
 
 if [ "$WITH_PYTHON2" = "ON" ]; then
-  ENV_LIST+="py27"
+  ENV_LIST+="py27,"
 fi
-if [ "$WITH_PYTHON3" = "ON" ]; then
-  ENV_LIST+="py3"
+if [ "$WITH_PYTHON3" = "3" ]; then
```

Comment:
Probably there is a good reason to change "WITH PYTHONX" variable domain from "ON/OFF" to "ON/OFF/3" but for me is not clear. Can you explain why this change improves the script? thx in advance

Reply:
@jmolmo Thanks for your comment! I just opened https://github.com/ceph/ceph/pull/28002 to address it. As to why I embarked on this project in the first place, see https://tracker.ceph.com/issues/39579